pkg/operator/configobservation/apiserver/observe_apiserver.go

```diff
@@ -134,6 +133,18 @@ func observeNamedCertificates(apiServer *configv1.APIServer, recorder events.Rec
 	namedCertificatesPath := []string{"servingInfo", "namedCertificates"}
 	resourceSyncRules := syncActionRules{}
 	var observedNamedCertificates []interface{}
+
+	// TODO add managed named cert info
+	observedNamedCertificates = append(observedNamedCertificates, map[string]interface{}{
+		"names":    []interface{}{"localhost", "127.0.0.1"},
+		"certFile": "/etc/kubernetes/static-pod-resources/secrets/serving-cert/tls.crt",
+		"keyFile":  "/etc/kubernetes/static-pod-resources/secrets/serving-cert/tls.key"})
+	observedNamedCertificates = append(observedNamedCertificates, map[string]interface{}{
+		"names":    []interface{}{"kubernetes", "kubernetes.default", "kubernetes.default.svc", "kubernetes.default.svc.cluster.local"},
```

Comment:
don't use the shortname references.  I think those people will die out eventually.

Reply:
Are "shortnames" all but "kubernetes.default.svc.cluster.local" in this list?